src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +55,33 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService) injector
+                .getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+                fastAndFurious.getId(), LocalDate.now()));
+
+        User newUser = new User();
+        newUser.setEmail("test@mail.com");
+        newUser.setPassword("password");
+        UserService userService = (UserService) injector
+                .getInstance(UserService.class);
+        userService.add(newUser);
+
+        ShoppingCartService cartService =
+                (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
+        cartService.registerNewShoppingCart(newUser);
+        cartService.addSession(tomorrowMovieSession, newUser);
+
+        ShoppingCart cart = cartService.getByUser(newUser);
+
+        OrderService orderService = (OrderService) injector
+                .getInstance(OrderService.class);
```

Comment:
Avoid explicit type casting by using a generic method in the `Injector` to prevent potential `ClassCastException`.